### PR TITLE
Fix drop replica syntax

### PR DESCRIPTION
**Pull request by jbajic**

### CODE REVIEW

File: docs/reference-guide/replication.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ two mains in the cluster.
 ## Limitations and potential features
 
 Currently, we do not support chained replicas, i.e. a replica can't have its
-own replica. When this feature becomes available, the user will be able to
```
Comment (jbajic) [inline docs/reference-guide/replication.md:94]: @g-despot this was automatically removed, if it's wrong can add it?

File: docs/reference-guide/replication.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ SHOW REPLICAS;
 To delete a replica, issue the following query:
 
 ```plaintext
-DROP REPLICA 'name';
+DROP REPLICA name;
```
Comment (jbajic) [inline docs/reference-guide/replication.md:74]: hmm, maybe this is better?
```suggestion
DROP REPLICA <name>;
```

Comment (antaljanosbenjamin) [reply docs/reference-guide/replication.md:74]: I definitely prefer the `<name>` form.

**Review by antonio2368 — APPROVED** [review]
This can go directly into master right?

**Review by kostasrim — APPROVED** [review]

**Review by antaljanosbenjamin — APPROVED** [review]

**Pull request merged**